patch/patches/icu.patch

```diff
@@ -29,3 +37,15 @@ index b3c12030..799dce2a 100644
        ['(OS=="linux" or OS=="freebsd" or OS=="openbsd" or OS=="solaris" \
           or OS=="netbsd" or OS=="mac" or OS=="android" or OS=="qnx") and \
          (target_arch=="arm" or target_arch=="ia32" or \
+@@ -171,9 +183,9 @@
+                  [ 'OS == "mac" or OS == "ios" or '
+                    '((OS == "android" or OS == "qnx") and '
+                    '_toolset == "host" and host_os == "mac")', {
+-                   'action': ['python', '<@(_inputs)', '<@(_outputs)', '--mac'],
++                   'action': ['<(python)', '<@(_inputs)', '<@(_outputs)', '--mac'],
+                  } , {
+-                   'action': ['python', '<@(_inputs)', '<@(_outputs)'],
++                   'action': ['<(python)', '<@(_inputs)', '<@(_outputs)'],
```

Comment:
https://chromium-review.googlesource.com/c/chromium/deps/icu/+/4392866/1
This looks like it can be upstreamed